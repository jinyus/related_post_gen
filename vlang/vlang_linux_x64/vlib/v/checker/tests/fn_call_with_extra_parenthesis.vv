fn doit(i int) {
}

fn main() {
	main.doit()(1)
}
