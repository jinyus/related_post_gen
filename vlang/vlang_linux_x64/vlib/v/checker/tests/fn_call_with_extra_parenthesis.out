vlib/v/checker/tests/fn_call_with_extra_parenthesis.vv:5:7: error: expected 1 arguments, but got 0
    3 |
    4 | fn main() {
    5 |     main.doit()(1)
      |          ~~~~~~
    6 | }
